logstf/logstf.sp

```diff
@@ -458,51 +499,60 @@ public Action:Command_say(client, args) {
 	return Plugin_Continue;
 }
 
-public QueryConVar_DisableHtmlMotd(QueryCookie:cookie, client, ConVarQueryResult:result, const String:cvarName[], const String:cvarValue[]) {
+public void QueryConVar_DisableHtmlMotd(QueryCookie cookie, int client, ConVarQueryResult result, const char[] cvarName, const char[] cvarValue) {
 	if (!IsClientValid(client))
 		return;
 	
 	if (result == ConVarQuery_Okay) {
 		if (StringToInt(cvarValue) != 0) {
-			decl String:nickname[32];
+			char nickname[32];
 			GetClientName(client, nickname, sizeof(nickname));
 			
 			CPrintToChat(client, "%s%s%s", "{lightgreen}[LogsTF] {default}", nickname, ": To see logs in-game, you need to set: {aqua}cl_disablehtmlmotd 0");
 			return;
 		}
 	}
 	
-	new Float:waitTime = 0.3;
+	float waitTime = 0.3;
 	waitTime -= GetTickedTime() - g_fSSTime[client];
 	if (waitTime <= 0.0)
-		waitTime = 0.01;
+	{
+		// sourcemod timers are at minimum 0.1 seconds
+		// -sappho
+		waitTime = 0.1;
```

Comment:
ftr you might also do a RequestFrame here, maybe?

Reply:
I don't want to handle two different cases - one for `<= 0.0` and one for `> 0`. Waiting 0.1 is fine by me.